### PR TITLE
Improve Bluetooth HCI Command packet definition

### PR DESCRIPTION
**Pull request by benquike**
- Divide opcode into ogf and ocf following Core Spec
- Redefine existing HCI commands with new format

**Checklist:**

-   [x] If you are new to Scapy: I have checked [CONTRIBUTING.md](https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md) (esp. section submitting-pull-requests)
-   [x] I squashed commits belonging together
-   [x] I added unit tests or explained why they are not relevant
-   [x] I executed the regression tests (using `cd test && ./run_tests` or `tox`)
-   [ ] If the PR is still not finished, please create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)

The ogf and ocf fields in the Bluetooth  HCI Command Header are grouped together, 
making it difficult to check these individual fields and mapping them back to packet definitions in
Core spec.

This change divide the opcode field into ogf and ocf bitfields following the defitions in core spec.

### CODE REVIEW

File: test/scapy/layers/bluetooth.uts [hunk 1]
```diff
@@ -4,26 +4,26 @@
...
..._Connect_Accept_Timeout()/HCI_Cmd_Disconnect()/HCI_Cmd_LE_Connection_Update()/HCI_Cmd_LE_Create_Connection()/HCI_Cmd_LE_Create_Connection_Cancel()/HCI_Cmd_LE_Host_Supported()/HCI_Cmd_LE_Long_Term_Key_Request_Negative_Reply()/HCI_Cmd_LE_Long_Term_Key_Request_Reply()/HCI_Cmd_LE_Read_Buffer_Size()/HCI_Cmd_LE_Set_Advertise_Enable()/HCI_Cmd_LE_Set_Advertising_Data()/HCI_Cmd_LE_Set_Advertising_Parameters()/HCI_Cmd_LE_Set_Random_Address()/HCI_Cmd_LE_Set_Scan_Enable()/HCI_Cmd_LE_Set_Scan_Parameters()/HCI_Cmd_LE_Start_Encryption_Request()/HCI_Cmd_Authentication_Requested()/HCI_Cmd_Link_Key_Request_Reply()/HCI_Cmd_Read_BD_Addr()/HCI_Cmd_Remote_Name_Request()/HCI_Cmd_Reset()/HCI_Cmd_Set_Connection_Encryption()/HCI_Cmd_Set_Event_Filter()/HCI_Cmd_Set_Event_Mask()/HCI_Command_Hdr()/HCI_Event_Command_Complete()/HCI_Event_Command_Status()/HCI_Event_Connect_Complete()/HCI_Event_Disconnection_Complete()/HCI_Event_Encryption_Change()/HCI_Event_Remote_Name_Request_Complete()/HCI_Event_Hdr()/HCI_Event_LE_Meta()/HCI_Event_Number_Of_Completed_Packets()/HCI_Hdr()/HCI_LE_Meta_Advertising_Reports()/HCI_LE_Meta_Connection_Complete()/HCI_LE_Meta_Connection_Update_Complete()/HCI_LE_Meta_Long_Term_Key_Request()
```
[the remark this answers is not in the document]
Comment (benquike) [reply test/scapy/layers/bluetooth.uts:7]: Done.

[the remark this answers is not in the document]
Comment (benquike) [reply test/scapy/layers/bluetooth.uts:7]: @gpotter2 can you please have a look at this change?

**Comment by gpotter2** [comment]
You have quite a bunch of failing tests. Could you fix those? Thanks

**Comment by gpotter2** [comment]
Please do not run an auto-formatter. This brings many un-necessary changes to the PR making it harder to review, and breaking the history. Could you revert this commit and only fix the formatting of your own changes? Thanks

**Comment by benquike** [comment]
@gpotter2 Can you please have a look?

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**